20xt/src/baja_proto/proto/baja_live_comm.proto

```diff
@@ -75,11 +80,20 @@ message CarState {
     optional bool rollover_status = 2;
 }
 
-message LiveComm {
+message RTKCorrection {
+    bytes rtk_correction = 1;
+}
+
+message Observation {
     optional GPS gps = 1;
```

Comment:
Can we change this to "one_of" instead of optional?

Additionally, make the timestamp non optional

Reply:
Making the timestamp non-optional is fine for the mqtt_client